package.json

```diff
@@ -2,29 +2,42 @@
   "name": "crud-car-registration",
   "version": "1.0.0",
   "description": "",
-  "main": "./src/server.ts",
+  "main": "./src/index.ts",
```

Comment:
Orientation: only to remember that one branch is specific to make specific things. If you working with Models, here in this branch you should not work on package.json.